kedro/extras/transformers/README.md

```diff
@@ -0,0 +1,6 @@
+# Information regarding transformers in Kedro
+
+# Time profiler
+This section outlines the prupose and functionality of the "time profiler" as a transformer. 
+By importing "AbstractTransformer", time profiler enables any contributors to track and log the running time for loading any dataset, as well as saving any calls.
+# Creator Name: TBD

```

Comment:
This doesn't need to be a header, and if it's TBD, we can probably just omit it for now :)